dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZookeeperCachedOperator.java

```diff
@@ -83,6 +83,7 @@ public void close() {
         try {
             Thread.sleep(500);
         } catch (InterruptedException ignore) {
+            Thread.currentThread().interrupt();
```

Comment:
why not simply remove the catch block ?

Reply:
> why not simply remove the catch block ?

Thank you very much for your suggestion, I will finish it later